### PR TITLE
Add status command

### PR DESCRIPTION
**Pull request by sniperking1234**
Signed-off-by: Chen Zhengwei <chenzhengwei@inspur.com>

## Which problem is this PR solving?
- Resolves https://github.com/jaegertracing/jaeger/issues/2361

## Short description of the changes
- Add status command. 
Usage: `<command> status` checks the localhost and the port of the corresponding component.
`<command> status --status.http.host-port` checks for a specific port or URL.

### CODE REVIEW

File: cmd/status/command_test.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+	err := cmd.Execute()
+	assert.NoError(t, err)
+}
+
+func TestOnlyPortConfig(t *testing.T) {
+	ts := httptest.NewServer(http.HandlerFunc(readyHandler))
+	defer ts.Close()
+	v := viper.New()
+	cmd := Command(v, 80)
+	cmd.ParseFlags([]string{"--status.http.host-port=" + strings.TrimPrefix(ts.URL, "http://127.0.0.1")})
```
Comment (yurishkuro) [inline cmd/status/command_test.go:52]: not sure if you're guaranteed to get URL having the exact 127.0.0.1 host (e.g. you could get IPv6 version instead). I'd rather read the port number as the part after the last `:`

**Review by yurishkuro — COMMENTED** [review]
look great, aside from a small nit in the test!

File: cmd/status/command_test.go (hunk 1 above)
Comment (sniperking1234) [reply cmd/status/command_test.go]: Done.

**Review by yurishkuro — APPROVED** [review]
Thanks!

**Pull request merged**